sections/main-product.liquid

```diff
@@ -180,16 +178,14 @@
                       {%- endif -%}
                     {%- else -%}
                       {%- if product.selected_or_first_available_variant.inventory_policy == 'continue' -%}
-                        <svg width="15" height="15" aria-hidden="true">
-                          <circle cx="7.5" cy="7.5" r="7.5" fill="rgb(62,214,96, 0.3)"/>
-                          <circle cx="7.5" cy="7.5" r="5" stroke="rgb(255, 255, 255)" stroke-width="1" fill="rgb(62,214,96)"/>
-                        </svg>
+                        <span style="color: rgb(62,214,96, 0.3)">
```

Comment:
```suggestion
                        <span style="color: rgb(62,214,96);display: flex;align-items: center;">
```

Adjusting here because:
- We need to center vertically the status circle
- Considering we can't apply an opacity to the outline, I suggest to remove the opacity entirely, I'll adapt the svg details on the file with my recommendations _(unless there is an alternative to keep the aesthetic of today, otherwise I am good.)_

Reply:
We can probably target it with CSS.  Lemme poke at it a bit.

Reply:
Ok let me know! If it works, we can ignore my suggestion to change the svg details from this comment https://github.com/Shopify/dawn/pull/3582#discussion_r1715880411

Reply:
Looks good color wise, just missing the vertical alignment to adjust! 🙌

Reply:
I will do that part in the Phase 3 PR because the way they're aligned changes there.